docs/source/library/Library.rst

```diff
@@ -0,0 +1,17 @@
+Library
+=======
+
+.. toctree::
+    :maxdepth: 4
+
+
+    program
+    properties
+    read_program_settings
+    simple_simulation
+    cost_aware_synthesis
+    line_aware_synthesis
+    circuit
+    gate
+    gate_type
+    bitset
```

Comment:
That was not really what I intended with my previous comment. There probably shouldn't be that many individual files but their content should be unified in a handful of files (as proposed in the other comment)

Reply:
done 👍🏻